content/testimonials/0.anonymous.md

```diff
@@ -1,5 +1,5 @@
 ---
-username: "anonymous"
+username: "Anonymous"
```

Comment:
That's not what I meant though. I don't mind lower or upper here. Can be either.

The testimonial text itself as the start of each sentens should start with Uppercase.

```diff
- my partner
+ my partner
```

```diff
- we're
+ We are
```

Reply:
done.